srcpkgs/chatterino2/template

```diff
@@ -1,55 +1,52 @@
 changelog="https://chatterino.com/changelog"
-distfiles="https://github.com/fourtf/chatterino2/archive/v${version}.tar.gz
- https://github.com/pajlada/signals/archive/${_signals_commit}.tar.gz
+distfiles="https://github.com/Chatterino/chatterino2/archive/refs/tags/v${version}.tar.gz
+ https://github.com/Chatterino/libcommuni/archive/${_libcommuni_commit}.tar.gz
+ https://github.com/pajlada/serialize/archive/${_serialize_commit}.tar.gz
  https://github.com/pajlada/settings/archive/${_settings_commit}.tar.gz
- https://github.com/pajlada/humanize/archive/${_humanize_commit}.tar.gz
- https://github.com/pajlada/serialize/archive/${_serialize_commit}.tar.gz"
-checksum="6e3d64f319728ccf9fd09613df754108157e7d3b4148ee2da1a3209bf369be70
-e8755e68ff6fc39b4338d424ee9aed2ed5a8eab6eb37af69de2ff21b8dc94e66
-f387cc7eaf27dec462bc85a212619f8086167cf0b47e11825b8580d35a740dfe
-5a5cd8338fcb0a14a4bbe0840f83505d0444a9000956145e85e5550feeb744b0
-eb492b89e67f4ec632bd43f2701d53e355f1d1afbec3cd3af5891c1a6aaaa454"
+ https://github.com/pajlada/signals/archive/${_signals_commit}.tar.gz
+ https://github.com/Chatterino/qtkeychain/archive/${_qtkeychain_commit}.tar.gz
+ https://github.com/arsenm/sanitizers-cmake/archive/${_sanitizers_cmake_commit}.tar.gz"
+checksum="27d42138d48d807340fe4194b4cdada6183b0ff2127f24d39032b3ad1f66594c
+ ba2179eb43638366ca65214137a850a127fb93a36fcc54658649c34e8d6f7dc4
+ d0da2cebd5dfd7b20f7247f1e4fb6b8a866434db49316a067b6c798ec8217c63
+ 0b35eb297f0949f27b3c54bc66d8b0c9293102fb52838d12371a5fcc80c590e5
+ 7ffd43dbd05704af5bef64b0eae8c12174013d4c7105472019ef381ff82c0a46
+ 4fdd7fd8bbae4fd7a505c917facc6e6b838850cd4f016c30fae45535dbaab4b7
+ f9cf386638f455c5d2e7a835b95941201387d2531b8682942d59827663b58341"
 
 if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 	makedepends+=" libatomic-devel"
-	configure_args+=" LIBS+=-latomic"
+	LIBS="-latomic"
 fi
 
 post_extract() {
-	rmdir lib/signals
-	rmdir lib/settings
-	rmdir lib/humanize
-	rmdir lib/serialize
-	mv "${XBPS_BUILDDIR}/signals-${_signals_commit}/" lib/signals/
-	mv "${XBPS_BUILDDIR}/settings-${_settings_commit}/" lib/settings/
-	mv "${XBPS_BUILDDIR}/humanize-${_humanize_commit}/" lib/humanize/
-	mv "${XBPS_BUILDDIR}/serialize-${_serialize_commit}/" lib/serialize/
-}
-
-pre_configure() {
-	# it wraps libcommuni in a namespace but void's binaries don't
-	find \( -name '*.cpp' -o -name '*.hpp' \) \
-		-exec sed -i "{}" -e 's/Communi::/::/g' \;
+	rmdir lib/libcommuni lib/serialize lib/settings lib/signals lib/rapidjson \
+	      lib/websocketpp lib/qtkeychain cmake/sanitizers-cmake
+	mv "${XBPS_BUILDDIR}/libcommuni-${_libcommuni_commit}" "lib/libcommuni"
+	mv "${XBPS_BUILDDIR}/serialize-${_serialize_commit}" "lib/serialize"
+	mv "${XBPS_BUILDDIR}/settings-${_settings_commit}" "lib/settings"
+	mv "${XBPS_BUILDDIR}/signals-${_signals_commit}" "lib/signals"
+	mv "${XBPS_BUILDDIR}/qtkeychain-${_qtkeychain_commit}" "lib/qtkeychain"
+	mv "${XBPS_BUILDDIR}/sanitizers-cmake-${_sanitizers_cmake_commit}" "cmake/sanitizers-cmake"
```

Comment:
Can we use create_wrksrc to have this all contained in one directory that will be cleaned with xbps-src clean?

Reply:
That would require us to modify the patches accordingly to $build_wrksrc on every update :/ I would leave it as it is for now. Or do you know a better way?

Reply:
I also could set `build-wrksrc=chatterino2` and `mv "$pkgname-$version chatterino2`. Does that sound like a good solution?